src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -257,6 +257,12 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         return get_self<ControlCore>(_termControl->_core);
     }
 
+    Windows::UI::ViewManagement::AccessibilitySettings TermControl::_GetAccessibilitySettings()

```

Comment:
you can save a call to `AddRef` and `Release` every time by returning an `AccessibilitySettings&`.